### PR TITLE
Tweak new tools/example workflow description

### PR DESCRIPTION
**Pull request by stichbury**
## Description
Created this to make a few minor tweaks to the docs about the `kedro new`workflow where tools & examples are described. This is a preview of the page most impacted: https://kedro--3454.org.readthedocs.build/en/3454/get_started/new_project.html

## Development notes

* Added copy to reproduce what the console says to make it easier for a reader to match up the docs and practical experience
* Added some quickstart examples as a "cheatsheet"
* Attempted to explain the example projects a little more clearly as it's super-complicated at first sight.
* Few minor formatting issues

## Developer Certificate of Origin
We need all contributions to comply with the [Developer Certificate of Origin (DCO)](https://developercertificate.org/). All commits must be signed off by including a `Signed-off-by` line in the commit message. [See our wiki for guidance](https://github.com/kedro-org/kedro/wiki/Guidelines-for-contributing-developers/).

If your PR is blocked due to unsigned commits, then you must follow the instructions under "Rebase the branch" on the GitHub Checks page for your PR. This will retroactively add the sign-off to all unsigned commits and allow the DCO check to pass.

## Checklist

- [ ] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [ ] Signed off each commit with a [Developer Certificate of Origin (DCO)](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-the-commit-signoff-policy-for-your-repository)
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes
- [ ] Checked if this change will affect Kedro-Viz, and if so, communicated that with the Viz team

### CODE REVIEW

File: docs/source/get_started/new_project.md [hunk 1]
```diff
@@ -2,23 +2,22 @@
 
 There are several ways to create a new Kedro project. This page explains the flow to create a basic project using `kedro new` to output a project directory containing the basic files and subdirectories that make up a Kedro project.
 
-You can also create a new Kedro project with a starter that adds a set of code for a common project use case. [Starters are explained separately](../starters/starters.md) later in the documentation set and illustrated with the [spaceflights tutorial](../tutorial/tutorial_template.md).
+You can also create a new Kedro project with a starter that adds code for a common project use case. [Starters are explained separately](../starters/starters.md) later in the documentation set and the [spaceflights tutorial](../tutorial/tutorial_template.md) illustrates their use.
```
Comment (astrojuanlu) [inline docs/source/get_started/new_project.md:5]:
```suggestion
You can also create a new Kedro project with a starter that adds code for a common project use case. [Starters are explained separately](../starters/starters.md) and the [spaceflights tutorial](../tutorial/tutorial_template.md) illustrates their use.
```

?

File: docs/source/get_started/new_project.md [hunk 2]
```diff
@@ -28,29 +27,91 @@ So, if you enter "Get Started", the folder for the project (`repo_name`) is auto
...
+
+Tools
+1) Lint: Basic linting with Black and Ruff
+2) Test: Basic testing with pytest
+3) Log: Additional, environment-specific logging options
+4) Docs: A Sphinx documentation setup
+5) Data Folder: A folder structure for data management
+6) PySpark: Configuration for working with PySpark
+7) Kedro-Viz: Kedro's native visualisation tool
+```
```
Comment (astrojuanlu) [inline docs/source/get_started/new_project.md:43]: This is the output I got with `main`:

````suggestion
```text
Tools
1) Lint: Basic linting with Black and Ruff
2) Test: Basic testing with pytest
3) Log: Additional, environment-specific logging options
4) Docs: A Sphinx documentation setup
5) Data Folder: A folder structure for data management
6) PySpark: Configuration for working with PySpark
7) Kedro-Viz: Kedro's native visualisation tool

Which tools would you like to include in your project? [1-7/1,3/all/none]:
 (none): 
```
````

skipped a few intro lines:

```
Project Tools
=============
These optional tools can help you apply software engineering best practices.
To skip this step in future use --tools
To find out more: https://docs.kedro.org/en/stable/starters/new_project_tools.html
```

File: docs/source/get_started/new_project.md [hunk 3]
```diff
@@ -28,29 +27,91 @@ So, if you enter "Get Started", the folder for the project (`repo_name`) is auto
...
+Finally, the CLI offers the option to include starter example code in the project:
 
-* [Default Starter](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights-pandas) (`spaceflights-pandas`): Used when you select any combination of Linting, Testing, Custom Logging, Documentation, and Data Structure, excluding PySpark and Kedro Viz.
-* [PySpark Starter](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights-pyspark) (`spaceflights-pyspark`): Chosen when PySpark is selected with any other tools, except Kedro Viz.
-* [Kedro Viz Starter](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights-pandas-viz) (`spaceflights-pandas-viz`): Applicable when Kedro Viz is part of your selection, with any other tools, excluding PySpark.
-* [Full Feature Starter](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights-pyspark-viz) (`spaceflights-pyspark-viz`): This example is used when you select all available tools, including PySpark and Kedro Viz.
+```text
+Select whether you would like an example spaceflights pipeline included in your project.
+```
```
Comment (astrojuanlu) [inline docs/source/get_started/new_project.md:56]: This is what I got from `main`:

````suggestion
```text
Would you like to include an example pipeline? :
 (no): 
```
````

Again, excluding some intro lines:

```
Example Pipeline
================
Select whether you would like an example spaceflights pipeline included in your project.
To skip this step in the future use --example=y/n
To find out more: https://docs.kedro.org/en/stable/starters/new_project_tools.html
```

File: docs/source/get_started/new_project.md [hunk 4]
```diff
@@ -28,29 +27,91 @@ So, if you enter "Get Started", the folder for the project (`repo_name`) is auto
...
+
+* [Default spaceflights starter (`spaceflights-pandas`)](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights-pandas): Added if you selected any combination of linting, testing, custom logging, documentation, and data structure, unless you also selected PySpark or Kedro Viz.
+* [PySpark spaceflights starter (`spaceflights-pyspark`)](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights-pyspark): Added if you selected PySpark with any other tools, unless you also selected Kedro Viz.
+* [Kedro Viz spaceflights starter (`spaceflights-pandas-viz`)](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights-pandas-viz): Added if Kedro Viz was one of your tools choices, unless you also selected PySpark.
+* [Full feature spaceflights starter (`spaceflights-pyspark-viz`)](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights-pyspark-viz): Added if you selected all available tools, including PySpark and Kedro Viz.
 
 Each starter example is tailored to demonstrate the capabilities and integrations of the selected tools, offering a practical insight into how they can be utilised in your project.
 
+### Quickstart examples
```
Comment (astrojuanlu) [inline docs/source/get_started/new_project.md:67]: Really like this section 💯

File: docs/source/get_started/new_project.md [hunk 5]
```diff
@@ -28,29 +27,91 @@ So, if you enter "Get Started", the folder for the project (`repo_name`) is auto
...
+kedro new ⮐
+My-Project ⮐
+none ⮐
+no ⮐
+```
+
+You can also enter this in a single line as follows:
+
+```bash
+kedro new --name=My-Project --tools=none -example=n
```
Comment (astrojuanlu) [inline docs/source/get_started/new_project.md:81]:
```suggestion
kedro new --name=My-Project --tools=none --example=n
```

File: docs/source/get_started/new_project.md [hunk 6]
```diff
@@ -28,29 +27,91 @@ So, if you enter "Get Started", the folder for the project (`repo_name`) is auto
...
+kedro new ⮐
+spaceflights ⮐
+7 ⮐
+yes ⮐
+```
+
+You can also enter this in a single line as follows:
+
+```bash
+kedro new --name=spaceflights --tools=viz -example=y
```
Comment (astrojuanlu) [inline docs/source/get_started/new_project.md:96]:
```suggestion
kedro new --name=spaceflights --tools=viz --example=y
```

File: docs/source/get_started/new_project.md [hunk 7]
```diff
@@ -28,29 +27,91 @@ So, if you enter "Get Started", the folder for the project (`repo_name`) is auto
...
+kedro new ⮐
+testproject ⮐
+1,4,6 ⮐
+no ⮐
+```
+
+You can also enter this in a single line as follows:
+
+```bash
+kedro new --name=testproject --tools=Lint, Docs, PySpark -example=n
```
Comment (astrojuanlu) [inline docs/source/get_started/new_project.md:111]:
```suggestion
kedro new --name=testproject --tools=lint,docs,pyspark --example=n
```

Maybe let's make all `--tools` examples lowercase?

**Review by astrojuanlu — CHANGES_REQUESTED** [review]
Nice work @stichbury - I left some comments, mainly to correct some of the quickstart one-liners but also to make the walkthrough more aligned with the current result. There's a question about how much of the actual text we want to include.

**Comment by stichbury** [comment]
Thanks for the review @astrojuanlu -- all changes accepted.

**Review by astrojuanlu — APPROVED** [review]
LGTM 🚀

**Review by DimedS — APPROVED** [review]
Thanks, @stichbury . More clear now!

**Comment by stichbury** [comment]
I could merge this but would value any insight from @AhdraMeraliQB since she worked on these docs prior to the release, and has the best context. I'll hold off on a merge until she's had a chance to feed back.

**Review by AhdraMeraliQB — APPROVED** [review]
These look amazing! I really like the addition of "next" and "finally" to connect how the prompts flow together, and I love the Quickstart examples section! 🌟

**Pull request merged**